### DISCUSSION TITLE
Herd Windows Slow

### DISCUSSION THREAD

**Discussion by andrefelipe18** [beyondcode/herd-community#1087] [General]
When I run a project with herd on windows, everything is slow, downloading a package is slow, running tests is slow, even running phpstan

![image](https://github.com/user-attachments/assets/742e4ae5-914c-46d1-9c2f-e5b684bef35d)

When I run the same project on a computer without herd, the speed is normal

**Comment by davidtim** [comment]
I almost buy Pro version but just notice that it's very slow, just to create new project! now back to wsl

**Reply by mpociot** [reply]
Unfortunately this is not an issue with Herd but with PHP on windows itself. 
Adding the Herd site folder to the windows defender exclude list helps a bit though.
